src/renderer/contexts/SettingsContext.tsx

```diff
@@ -66,7 +66,9 @@ export const SettingsProvider = memo(
         // update theme
         const { setColorMode } = useColorMode();
         useEffect(() => {
-            setColorMode(settings.theme);
+            const [themeName, darkOrLight] = settings.theme.split('-');
+            setColorMode(darkOrLight);
```

Comment:
Q: Couldn't we "hack" the System theme back in if we called it `default-system` internally?

Reply:
That still would leave the user with no way to change _which_ theme it uses under system mode. We'd just have to use default and they'd have to be fine with that, unless we add extra settings

Reply:
I don't think that's a problem. If they don't like the theme we auto picked, they can just change the theme to something they like more.